### PR TITLE
Node Js week3 Homework

### PR DESCRIPTION
**Pull request by sayali05**

### CODE REVIEW

File: nodejs/week3/app.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.json());
+
+const apiRouter = express.Router();
+app.use("/api", apiRouter);
+
+const contactsAPIRouter = express.Router();
+apiRouter.use("/contacts", contactsAPIRouter);
```
Comment (YurijPaharev) [inline nodejs/week3/app.js:23]: Why creating apiRouter and not using it afterwards?

File: nodejs/week3/app.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  if ("sort" in req.query) {
+    const orderBy = req.query.sort.toString();
+    //console.log(orderBy);
+    if (orderBy.length > 0) {
+      // ?? is used to indicate the column name
+      query = query.orderByRaw("??", orderBy);
+    }
+  }
+
+  console.log("SQL", query.toSQL().sql);
```
Comment (YurijPaharev) [inline nodejs/week3/app.js:37]: Remove this log

**Review by YurijPaharev — CHANGES_REQUESTED** [review]
Nice work. Please apply changes stated in comments

File: nodejs/week3/app.js (hunk 2 above)
Comment (sayali05) [reply nodejs/week3/app.js:37]: sure i will update code as per your suggestion. Thank you.

**Pull request merged**